dts/bindings/pwm/nuvoton,npcx-pwm.yaml

```diff
@@ -18,11 +18,11 @@ properties:
         required: true
     pinctrl-names:
         required: true
-    drive-open-drain:
-        type: boolean
+    channel:
```

Comment:
`channel` is also a property of the `nuvoton,npcx-ps2-channel.yaml` node.  This won't break the `Z_PINCTRL_NPCX_HAS_DRIVE_PROP` macro, but I suggest creating a unique name in this node.  How about `pwm-channel`?  The open-drain support is already tightly coupled to the PWM driver.

Reply:
Done.